test/functional/wallet_descriptor.py

```diff
@@ -16,18 +16,22 @@ def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
         self.extra_args = [['-keypool=100']]
+        self.wallet_names = []
 
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
+        self.skip_if_no_sqlite()
```

Comment:
Is it necessary to call `self.skip_if_no_sqlite()` explicitly here, since it is already called in `skip_if_no_wallet(self)` if `--descriptors` is set (and looking at the additions to `test_runner.py` this seems to now be the case)?

Reply:
It's being done for people who just run it on the command line without `--descriptor`. This test always needs this check.